src/engraving/dom/score.cpp

```diff
@@ -427,6 +428,9 @@ void Score::setUpTempoMap()
         if (m->mmRest()) {
             m->mmRest()->moveTicks(diff);
         }
+        if (m->ticks() < m->timesig()) {
```

Comment:
Might still want to use `m->isAnacrusis()`

Reply:
isAnacrusis() not works as expected. 
```cpp
return irregular() && ticks() < Fraction::fromTicks(timeSig.ticksPerMeasure());
```
```irregular``` can be false if user make pickup bar after started the score an didn't check ```Exclude from measure count```
<img width="564" alt="image" src="https://github.com/user-attachments/assets/3b0a7116-e8ad-4c90-ac5e-9f7949afbf73">
But what anacrusis is, actually a measure where real length is less than timesig.
<img width="1228" alt="image" src="https://github.com/user-attachments/assets/14455cb2-a1b9-44bf-abd5-85184142a3e4">
Otherwise this is a notation mistake, which as far as I know not allowed by MuseScore

Reply:
Actually I see an anacrusis being an anacrusis only if it is a) shorter than the current time sig and b) is excluded from measure count